postgres-14/Dockerfile

```diff
@@ -12,16 +12,20 @@ LABEL maintainer="Citus Data https://citusdata.com" \
       org.label-schema.schema-version="1.0"
 
 ENV CITUS_VERSION ${VERSION}.citus-1
+#install postgresql
+RUN apt-get update && apt-get install -y lsb-release apt-utils wget curl gnupg2 && apt-get clean all \
+    && wget --quiet -O - https://www.postgresql.org/media/keys/ACCC4CF8.asc | apt-key add - \
+    && sh -c 'echo "deb http://apt.postgresql.org/pub/repos/apt $(lsb_release -cs)-pgdg main 14" > /etc/apt/sources.list.d/pgdg.list' \
+    && apt-get purge -y --auto-remove curl \
+    && rm -rf /var/lib/apt/lists/*
```

Comment:
I did not try to build the following on my local. Please review it before applying it.

I wanted to improve readability and remove unnecessary commands.

```suggestion
# install postgresql 14
RUN apt-get update \
    && apt-get install -y --no-install-recommends \
       apt-utils \
       gnupg2 \
       lsb-release \
       wget \
    && wget --quiet -O - https://www.postgresql.org/media/keys/ACCC4CF8.asc | apt-key add - \
    && echo "deb http://apt.postgresql.org/pub/repos/apt $(lsb_release -cs)-pgdg main 14" > /etc/apt/sources.list.d/pgdg.list \
    && rm -rf /var/lib/apt/lists/*
```

Reply:
Several questions:
1. Why do you need `sh -c` to execute the commands?
2. Why do you need `curl`?
3. Can you provide one package name per line?